strings_lab.js

```diff
@@ -0,0 +1,76 @@
+//1. console.log the first character of a string.
```

Comment:
Overall good demonstration of string properties and methods. I think you are fully capable of solving bonus questions so try them next time. Keep it up!